### PR TITLE
Change project structure

### PR DESCRIPTION
**Pull request by s-samusik**
This PR closes issue #17 
Added a separate  business logic layer. And xUnit test project for test this layer.

### CODE REVIEW

File: BookSharing.BLL/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
```
Comment (N0menIllisLegio) [inline BookSharing.BLL/Class1.cs:1]: Do we need this?

File: BookSharing.Tests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+using System;
```
Comment (N0menIllisLegio) [inline BookSharing.Tests/UnitTest1.cs:1]: And this

**Review by N0menIllisLegio — CHANGES_REQUESTED** [review]
Do we need empty files?

**Comment by s-samusik** [comment]
I created empty projects to show main structure of solution. These classes need to be implemented in tasks #19.

**Review by N0menIllisLegio — APPROVED** [review]

**Pull request merged**